nss.sql

```diff
@@ -0,0 +1,68 @@
+create schema nss_pgsql;
+
+create view nss_pgsql.groupmember as
+    select
+        name as username,
+        uid as gid
+    from public.passwd
+    union
+    select
+        name as usernames,
+        aux_groups.gid as gid
+        from public.passwd inner join public.aux_groups
+            on (passwd.uid = aux_groups.uid);
+grant select on nss_pgsql.groupmember to nss_pgsql;
```

Comment:
Would like to know what `to nss_pgsql` means as I assume it means a role, which AFAICT we do not create right now?